descriptors/ModuleDescriptor-template.json

```diff
@@ -302,6 +302,15 @@
             "orders-storage.po-lines.item.get",
             "orders-storage.po-lines.item.put"
           ]
+        },
+        {
+          "methods": [
+            "PUT"
+          ],
+          "pathPattern": "/orders/order-lines/fund-distributions/validate",
```

Comment:
nitpick: REST endpoints should have nouns instead of verbs so "validate" would not be ideal here. Actions such as this would extra data points to work and I doubt FOLIO endpoints follow this as well. That is why this is just FYI for now. It could be something like the below:
`
curl --location --request PUT '/orders/order-lines/fund-distributions' \
--header 'Content-Type: application/folio.orders.fundvalidate+json'
`
I doubt our RAMLs handle content types...

Reply:
Yes, agree that http method should be as verb and endpoint name - noun, but I have not yet met handing based on content type in RMB in the same way as provided in comment, but definitely it's a good point